data_store/transformation/towns_fund/tf_transform_r3.py

```diff
@@ -623,14 +617,7 @@ def extract_funding_data(df_input: pd.DataFrame, project_lookup: dict, reporting
     df_funding.drop(unused_mask.index, inplace=True)
 
     if fund_type == "HS":
-        # Round 3 collects up to H2 23/24, round 4 collects up to H1 24/25, round 5 collects up to H2 24/25
-        start_date_cut_off_mapping = {
-            3: datetime(2023, 10, 1),
-            4: datetime(2024, 4, 1),
-            5: datetime(2024, 10, 1),
-            6: datetime(2025, 4, 1),
-        }
-        start_date_cut_off = start_date_cut_off_mapping[reporting_round]
+        start_date_cut_off = datetime(2023, 10, 1)
```

Comment:
question: Could this theoretically be fed from one of the values on the reporting round table instead?